src/measure_theory/measure/measure_space.lean

```diff
@@ -1816,13 +1816,66 @@ begin
   exact eq_univ_of_subset (Union_subset_Union $ λ n, subset_to_measurable μ (s n)) hs
 end
 
+/-- Given two σ-finite measures `μ` and `ν`, `finite_spanning_setsin.add μ ν` is a sequence of
+pairwise disjoint, finite spanning sets in `{s | measurable_set s}` with respect to `μ + ν`. -/
+protected def finite_spanning_sets_in.add
```

Comment:
I am not sure I understand why you need this: if your goal is to prove `exists_eq_disjoint_finite_spanning_sets_in`, why not start from a spanning set for `μ + ν` (which is also sigma-finite), make it disjoint thanks to `finite_spanning_sets_in.disjointed`, and then argue that it is a spanning set both for mu and nu thanks to `finite_spanning_sets_in.of_le`?